sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -97,6 +97,7 @@ task integrationTest(type: Test) {
   outputs.upToDateWhen { false }
 
   include '**/*IT.class'
+  exclude '**/BigQueryIOReadIT.class'
```

Comment:
AIUI we don't want to exclude the entire test -- can we exclude only part of it?

Reply:
Unfortunately no. The 'exclude' here only works on file pattern so that we could only exclude the entire class. I found an incubating Gradle [TestFilter](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/TestFilter.html) that is able to execute specific test methods. But it doesn't work with the 'exclude' to skip test methods.

Reply:
https://discuss.gradle.org/t/how-to-exclude-a-single-test-method/20860